salt/modules/monit.py

```diff
@@ -149,3 +158,109 @@ def status(svc_name=''):
     else:
         ret = entries.get(svc_name, 'No such service')
     return ret
+
+
+def reload_():
+    '''
+    .. versionadded:: Boron
+
+    Reload monit configuration
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' monit.reload
+    '''
+    cmd = 'monit reload'
+    return not __salt__['cmd.retcode'](cmd, python_shell=False)
+
+
+def configtest():
+    '''
+    .. versionadded:: Boron
+
+    Test monit configuration syntax
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' monit.configtest
+    '''
+    ret = {}
+    cmd = 'monit -t'
+    out = __salt__['cmd.run_all'](cmd)
+
+    if out['retcode'] != 0:
+        ret['comment'] = 'Syntax Error'
+        ret['stderr'] = out['stderr']
+        ret['result'] = False
+        return ret
+
+    ret['comment'] = 'Syntax OK'
+    ret['stdout'] = out['stdout']
+    ret['result'] = True
+    return ret
+
+
+def version():
+    '''
+    .. versionadded:: Boron
+
+    Return version from monit -V
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' monit.version
+    '''
+    cmd = 'monit -V'
+    out = __salt__['cmd.run'](cmd).splitlines()
+    ret = out[0].split()
+    return ret[-1]
+
+
+def id_(reset=False):
+    '''
+    .. versionadded:: Boron
+
+    Return monit unique id.
+
+    reset : False
+        Reset current id and generate a new id when it's True.
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' monit.id [reset=True]
+    '''
+    if reset:
+        id_pattern = re.compile(r'Monit id (?P<id>[^ ]+)')
+        cmd = 'echo y|monit -r'
+        out = __salt__['cmd.run_all'](cmd, python_shell=True)
+        ret = id_pattern.search(out['stdout']).group('id')
+        return ret if ret else False
+    else:
+        cmd = 'monit -i'
+        out = __salt__['cmd.run'](cmd)
+        ret = out.split(':')[-1].strip()
+    return ret
```

Comment:
for consistency, you could have that return in the else or set ret to False if it's None in the if block and let that last line return what ever is in ret ...